playwright/tests/auth.setup.ts

```diff
@@ -1,178 +1,94 @@
-    customerWebhooksPermissionsFile,
-  );
-});
-setup("authenticate as user with gift cards permissions", async ({ page }) => {
-  const loginPage = new LoginPage(page);
-  await loginPage.loginAndSetStorageState(
-    USER_PERMISSION.giftCard,
-    process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
-    page,
-    giftCardsPermissionsFile,
-  );
-});
-setup(
-  "authenticate as user with content aka page permissions",
-  async ({ page }) => {
-    const loginPage = new LoginPage(page);
-    await loginPage.loginAndSetStorageState(
-      USER_PERMISSION.page,
-      process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
-      page,
-      contentPermissionsFile,
-    );
-  },
-);
-setup("authenticate as user with plugins permissions", async ({ page }) => {
-  const loginPage = new LoginPage(page);
-  await loginPage.loginAndSetStorageState(
-    USER_PERMISSION.plugin,
-    process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
-    page,
-    pluginPermissionsFile,
-  );
-});
-setup(
-  "authenticate as user with product type permissions",
-  async ({ page }) => {
-    const loginPage = new LoginPage(page);
-    await loginPage.loginAndSetStorageState(
-      USER_PERMISSION.productTypeAndAttribute,
-      process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
-      page,
-      productTypePermissionsFile,
-    );
-  },
-);
-setup("authenticate as user with settings permissions", async ({ page }) => {
-  const loginPage = new LoginPage(page);
-  await loginPage.loginAndSetStorageState(
-    USER_PERMISSION.settings,
-    process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
-    page,
-    settingsPermissionsFile,
-  );
-});
-setup(
-  "authenticate as user with staff member permissions",
-  async ({ page }) => {
-    const loginPage = new LoginPage(page);
-    await loginPage.loginAndSetStorageState(
-      USER_PERMISSION.staff,
-      process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
-      page,
-      staffMemberPermissionsFile,
-    );
-  },
-);
-setup("authenticate as user with shipping permissions", async ({ page }) => {
-  const loginPage = new LoginPage(page);
-  await loginPage.loginAndSetStorageState(
-    USER_PERMISSION.shipping,
-    process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
-    page,
-    shippingPermissionsFile,
-  );
-});
-setup("authenticate as user with translation permissions", async ({ page }) => {
-  const loginPage = new LoginPage(page);
-  await loginPage.loginAndSetStorageState(
-    USER_PERMISSION.translations,
-    process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
-    page,
-    translationPermissionsFile,
-  );
-});
-setup("authenticate as user with product permissions", async ({ page }) => {
-  const loginPage = new LoginPage(page);
-  await loginPage.loginAndSetStorageState(
-    USER_PERMISSION.product,
-    process.env.E2E_PERMISSIONS_USERS_PASSWORD!,
-    page,
-    productPermissionsFile,
-  );
+  return {
+  "cookies": [
+    {
+      "name": "refreshToken",
+      "value": refreshToken,
+      "domain": "automation-dashboard.staging.saleor.cloud",
```

Comment:
Why it is hardcoded? It want be easier to just use .storageState() on APIRequestContext. There is already created cookie, the only thing to add in order to login is _saleorRefreshToken to local storage and save in file. This way we woulod not need to hardcode domain or count expires for this cookie